bugzilla2gitlab/models.py

```diff
@@ -44,17 +44,16 @@ def save(self):
             comment.save()
 
         # close the issue in GitLab, if it is resolved in Bugzilla
-        if self.issue.status == "RESOLVED":
+        if self.issue.status in ["RESOLVED", "VERIFIED", "CLOSED"]:
```

Comment:
Since there exists the possibility for many bugzilla statuses that map to "Closed" in gitlab, I'd like to see this as a configuration variable in `defaults.yml`

Reply:
Done